css-sizing-4/Overview.bs

```diff
@@ -543,32 +543,22 @@ Overriding Contained Intrinsic Sizes: the 'contain-intrinsic-*' properties</h3>
 	would still size according to the specified grid,
 	usually ending up with a larger content size than specified.
 
-	Values are defined as:
-
 	<dl dfn-type=value dfn-for="contain-intrinsic-width, contain-intrinsic-height, contain-intrinsic-block-size, contain-intrinsic-inline-size, contain-intrinsic-size">
-		: <dfn>none</dfn>
-		:: The corresponding axis does not have an [=explicit intrinsic inner size=].
+		: <dfn>auto</dfn>? [ <dfn>none</dfn> | <<length>> ]
+		:: If ''contain-intrinsic-size/auto'' is specified
+			and the element has a [=last remembered size=]
+			and is currently [=skipping its contents=],
+			its [=explicit intrinsic inner size=] is the [=last remembered size=].
 
-		: <dfn><<length>></dfn>
-		:: The corresponding axis has an [=explicit intrinsic inner size=] of the specified <<length>>.
+			Otherwise, if ''contain-intrinsic-size/none'' is specified,
+			the corresponding axis does not have an [=explicit intrinsic inner size=].
+
+			Otherwise,
+			the corresponding axis has an [=explicit intrinsic inner size=] of the specified <<length>>.
```

Comment:
```suggestion
			Otherwise,
			the corresponding axis either doesn't have an [=explicit intrinsic inner size=]
			(if ''contain-intrinsic-size/none'' is specified)
			or has an [=explicit intrinsic inner size=] of the specified <<length>>.
```